File: platform/platform-bukkit/src/main/java/taboolib/platform/BukkitWorldGenerator.java
package taboolib.platform;

import org.bukkit.generator.ChunkGenerator;
import org.jetbrains.annotations.Nullable;

/**
 * TabooLib
 * taboolib.platform.BukkitWorldGenerator
 *
 * @author 坏黑
 * @since 2024/1/26 16:23
 */
public interface BukkitWorldGenerator {

    @Nullable
    ChunkGenerator getDefaultWorldGenerator(String worldName, @Nullable String name);
}

File: platform-bukkit/src/main/kotlin/taboolib/platform/BukkitWorldGenerator.kt
package taboolib.platform

import org.bukkit.generator.ChunkGenerator

/**
 * TabooLib
 * taboolib.platform.BukkitWorldGenerator
 *
 * @author sky
 * @since 2021/6/16 9:39 下午
 */
interface BukkitWorldGenerator {

    fun getDefaultWorldGenerator(worldName: String, name: String?): ChunkGenerator?
}